src/utils.ts

```diff
@@ -347,10 +347,17 @@ export function maybeCallback<T>(
     return promise;
   }
 
-  promise.then(
-    result => callback(undefined, result),
-    error => callback(error)
-  );
+  promise
+    .then(
+      result => callback(undefined, result),
+      error => callback(error)
+    )
+    .catch(() => {
```

Comment:
Can we at least log something here so the user at least knows we are swallowing the error?

Reply:
I don't think we should add logs here, at least not with a `console.log`.  But I can file a ticket in the structured logging epic to add logging to places we may be suppressing errors?  There's a list of potential locations in the ticket.

Reply:
I guess I'd like to think what others think since it feels strange to me we'd swallow exceptions at this point (everything going through `executeOperation` and not notify in some form). I understand swallowing elsewhere without notifying (like `endSessions` in `close`), but `executeOperation` is a critical path for the driver operations.

Reply:
What is the error being observed that is causing the issue for your other work? 

I'm wondering if we wouldn't be better off correcting that "throw" to be properly sent to the callback, rather than introducing this more global silencing to all our maybeCallack usage.